fineract-e2e-tests-runner/src/test/resources/features/LoanDownPayment.feature

```diff
@@ -1111,8 +1111,8 @@ Feature: Loan DownPayment
       | 2  | 31   | 01 February 2022 | 15 February 2022 | 500.0           | 250.0         | 0.0      | 0.0  | 0.0       | 250.0 | 250.0 | 0.0        | 250.0 | 0.0         |
       |    |      | 15 February 2022 |                  | 1000.0          |               |          | 0.0  |           | 0.0   | 0.0   |            |       |             |
       | 3  | 0    | 15 February 2022 |                  | 1250.0          | 250.0         | 0.0      | 0.0  | 0.0       | 250.0 | 0.0   | 0.0        | 0.0   | 250.0       |
-      | 4  | 28   | 01 March 2022    |                  | 750.0           | 500.0         | 0.0      | 0.0  | 0.0       | 500.0 | 0.0   | 0.0        | 0.0   | 500.0       |
-      | 5  | 31   | 01 April 2022    |                  | 0.0             | 750.0         | 0.0      | 0.0  | 0.0       | 750.0 | 0.0   | 0.0        | 0.0   | 750.0       |
+      | 4  | 28   | 01 March 2022    |                  | 625.0           | 625.0         | 0.0      | 0.0  | 0.0       | 625.0 | 0.0   | 0.0        | 0.0   | 625.0       |
```

Comment:
I am afraid something here is still incorrect. The EMI should be 250 for every installment. On cumulative loans it doesnt really matter on which installment the disbursement occurred, it is divided across every period anyway.